lib/pagerduty.rb

```diff
@@ -68,7 +84,11 @@ def trigger(description, options = {})
   #
   def get_incident(incident_key)
     fail ArgumentError, "incident_key is nil" if incident_key.nil?
-    PagerdutyIncident.new service_key, incident_key
+    PagerdutyIncident.new(
+      service_key,
+      incident_key,
+      transport: @transport,
```

Comment:
We want the configuration to be used by all HTTP calls to pager duty, so we need to pass the transport to all created `PagerdutyIncident`s. This'll mean the `acknowledge` and `resolve` method calls will use the proxy.